tests/system/README.md

```diff
@@ -4,7 +4,25 @@ Wazuh - System quality assurance automation templates
 
 ## Setting up a test environment
 
-To run these tests we need to use a **Linux** machine and install the following tools:
+To run these tests we need to use a **Linux** machine.
+
+In the event that the tests are to be executed using AWS EC2 instances:
+
+### EC2 requirements for system testing.
+
+| Environment                  | EC2                                       |
+|------------------------------|-------------------------------------------|
+|Basic_cluster                 |Ubuntu 22.04.2 LTS `C5`.`XLarge` `15GB` HD |
+|Big_cluster_40_agents         |Ubuntu 22.04.2 LTS T3.Large `60GB` HD      |
+|Agentless_cluster             |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Four_manager_disconnected_node|Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|One_manager_agent             |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Manager_agent                 |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Enrollment_cluster            |Ubuntu 22.04.2 LTS T3.Large 30GB HD        | 
+| Basic_environment            |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
```

Comment:
AWS does not provide HD but HDD, also remove the extra space in line 33 between | and Basic

Why the 60GB is encapsulated between `` and 30GB no? The same goes for the type of instance.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4536/commits/ebbcdb5657a2c91573e29a50775b12441c2473bd